js/everything.js

```diff
@@ -427,6 +428,40 @@ function anchorURLs(text) {
     return text.replace(re, "<a href=\"$1\" target=\"_blank\">$1</a>");
 }
 
+function anchorTimestamps(text, videoID) {
+    /* RegEx to match
+      hh:mm:ss
+      h:mm:ss
+      mm:ss
+      m:ss
+    and wraps the timestamps in <a> tags
+    RegEx explanation:
+    ((?:[0-5]\d|\d|) either the string is "colon 00-59" or "0-9" or "blank"
+    (?:\d|\:[0-5]\d) either the string is "colon 0-9" or "colon 00-59"
+    (?:$|\:[0-5]\d)) either the string ends or is a a number between 00-59
+    */
+    var re = /((?:[0-5]\d|\d|)(?:\d|\:[0-5]\d)(?:$|\:[0-5]\d))/g;
+    return text.replace(re, function(match) {
+        return "<a href=\"" + makeListenURL(videoID, convertTimestamp(match)) + "\">" + match + "</a>";
+    });
+}
+
+function convertTimestamp(timestamp) {
+    var seconds = 0;
+    var minutes = 0;
+    var hours = 0;
+    var timeComponents = timestamp.split(":");
+    if (timeComponents.length === 3) {
+        hours = parseInt(timeComponents[0], 10) * 60 * 60; // convert hours to seocnds
+        minutes = parseInt(timeComponents[1], 10) * 60; // convert minutes to seconds
+        seconds = parseInt(timeComponents[2], 10); // add remaining seconds
+    } else {
+        minutes = parseInt(timeComponents[0], 10) * 60; // convert minutes to sec§onds
```

Comment:
I just noticed you have a funny character in the comment here`§`. Once that's fixed I'll do some manual testing before merging